edk2toolext/versioninfo/versioninfo_tool.py

```diff
@@ -0,0 +1,67 @@
+# @file versioninfo_tool.py
+# This module contains the CLI interface for easily creating VERSIONINFO resources
+# from json files.
+#
+##
+# Copyright (C) Microsoft Corporation
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+
+import os
+import sys
+import argparse
+import json
+from edk2toolext.versioninfo.versioninfo_helper import PEObject, VERSIONINFOGenerator
```

Comment:
I don't love the name VersionInfo, thoughts @spbrogan? I feel like it would get mixed up with version aggregator.

Reply:
@spbrogan @gatys Any thoughts on this before we merge this in?

Reply:
it is really binary info, image info, PE info, resource info?? It holds more than version.  Not sure if that would make this better.